Lib/stat.py

```diff
@@ -111,18 +112,31 @@ def S_ISWHT(mode):
 
 # Names for file flags
 
+if sys.platform == "darwin":
+    # Group of bits in st_flags, specific to Apple platforms.
+
+    UF_SETTABLE = 0xffff      # owner settable flags
+    SF_SUPPORTED = 0x9f0000   # superuser supported flags
+    SF_SETTABLE  = 0x3ffff000 # superuser settable flags
```

Comment:
It is suspicious that `UF_SETTABLE & SF_SETTABLE != 0`.

Reply:
I intend to add a test for macOS that tests against the actual platform values for every constant in `stat`, even if that might not survive review.

Reply:
> It is suspicious that `UF_SETTABLE & SF_SETTABLE != 0`.

This was a copy&paste error as well.